index.html

```diff
@@ -672,47 +549,49 @@ <h5>
               power saving non-functional requirement</a>. For example, the
               <a>user agent</a> might choose not to run the monitoring
               algorithm when the <a>global set of availability callbacks</a> is
-              empty, when every page that has <a data-lt="media element">media
-              elements</a> with non-empty <a>set of availability callbacks</a>
-              is in the background.
+              empty, when every page that has <a>media elements</a> with non-empty
+              <a>set of availability callbacks</a> is in the background.
             </p>
             <p>
               Some <a>remote playback devices</a> may only be able to play a
-              subset of <a data-link-for="media resource">media resources</a>
+              subset of <a data-cite="HTML#media-resource">media resources</a>
               because of functional, security or hardware limitations. Examples
               are set-top boxes, smart TVs or networked speakers capable of
               rendering only certain formats of video and/or audio. We say that
               such a device is a <dfn>compatible remote playback device</dfn>
-              for a <a>media resource</a> if the <a>user agent</a> can
-              reasonably guarantee that the remote playback of the media
-              specified by the resource will succeed on that device.
+              for a <a data-cite="HTML#media-resource">media resource</a>
+              if the <a>user agent</a> can reasonably guarantee that the
+              remote playback of the media specified by the resource will
+              succeed on that device.
             </p>
             <p class="note">
-              The <a>user agent</a> can use the <a><code>srclang</code></a>
-              attribute of the <a><code>track</code></a> element as a hint of
+              The <a>user agent</a> can use the <code>{{HTMLTrackElement/srclang}}</code>
+              attribute of the <code>[^track^]</code> element as a hint of
```

Comment:
Are those changes from elements to markdown required? (here and other places in this pull request)

Reply:
No, but it's a more simplified way ReSpec provided, you can search your references via https://respec.org/xref/?term=track.